app/controllers/authentication_controller.rb

```diff
@@ -0,0 +1,16 @@
+class AuthenticationController < ApplicationController
+  # return auth token once user is authenticated
+  skip_before_action :authorize_request, only: :authenticate
+  protect_from_forgery unless: -> { request.format.json? }
+
+  def authenticate
+    auth_token = AuthenticateUser.new(auth_params[:email], auth_params[:password]).call
+    json_response(auth_token: auth_token)
+  end
+
+  private
+
+  def auth_params
+    params.permit(:email, :password)
+  end
+end
```

Comment:
- [ ] Kindly, return the response in the JSON format after sending a request.

![api3](https://user-images.githubusercontent.com/55635977/149814040-1330a9dd-6b06-48c3-8e47-9c9bc87a2630.png)